rosdep/python.yaml

```diff
@@ -233,6 +233,15 @@ paramiko:
       packages: [paramiko]
   rhel: [python-paramiko]
   ubuntu: [python-paramiko]
+pcg-gazebo-pip:
```

Comment:
Could you please rename (and therefore relocate) this package to `python3-pcg-gazebo-pip` to conform to the upcoming contribution guidelines in #23836.

Reply:
Hello @nuclearsandwich, thanks for the tip, I had not seen that pull request. I added the tags for both 2.x and 3.x versions since it is available for both. Is the duplication OK? Python 2.7 is EOL but I don't know how to set these tags any other way for systems that still are using 2.7.

Reply:
I was about to speak out against duplicate keys but @tfoote is in favor based on https://github.com/ros/rosdistro/pull/23836#discussion_r382871821